templates/contest/official-ranking-table.html

```diff
@@ -0,0 +1,36 @@
+<thead>
+<tr>
+    <th class="header rank">{{ rank_header or _("Rank") }}</th>
+    <th class="header">{{ _('Username') }}</th>
+    <th class="header username">{{ _('Full Name') }}</th>
+    {% for problem in problems %}
+        <th class="points header"><a href="{{ url('contest_ranked_submissions', contest.key, problem.problem.code) }}">
+            {{- contest.get_label_for_problem(loop.index0) }}
+            <div class="point-denominator">{{ problem.points }}</div>
```

Comment:
The order of problems in the CSV file and in the contest are not the same, so put links in this header will lead to confusion. I think we should simply add put the name of the problem (which gets from the CSV file).